Sources/NIO/Linux.swift

```diff
@@ -15,6 +15,7 @@
 // This is a companion to System.swift that provides only Linux specials: either things that exist
 // only on Linux, or things that have Linux-specific extensions.
 import CNIOLinux
+import Foundation
```

Comment:
SwiftNIO doesn't use `Foundation` and must continue not to. Would you mind removing this import?